intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/verifiers/SimpleVerifiers.kt

```diff
@@ -0,0 +1,22 @@
+package com.jetbrains.plugin.structure.intellij.verifiers
```

Comment:
Maybe `SimpleValidators` if there are **validate**PropertyLength and **validate**NewLines?

Reply:
There is a slight confusion in terminology between _validation_ and _verification_. Despite the standard _validation_ (as in Java Validation API), due to the popularity in the codebase, I am in favour of `verifyPropertyLength`.